### PR TITLE
Update java-openliberty-gradle devfile

### PR DESCRIPTION
**Pull request by dkwon17**
### What does this PR do?:
This PR updates the java-openliberty-gradle sample:
1. Update schemaversion to 2.2.2
2. Update endpoint protocol to https

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

**Review by thepetk — CHANGES_REQUESTED** [review]
@dkwon17 I think here we should also bump up the version of the devfile

https://github.com/devfile/registry/blob/ae8898b271d1c335492c9cb91d35b16a6b070ade/stacks/java-openliberty-gradle/devfile.yaml#L33

I'd suggest bumping up to `0.5.0`

File: stacks/java-openliberty-gradle/devfile.yaml
Comment (thepetk) [inline stacks/java-openliberty-gradle/devfile.yaml:55]:
```suggestion
          name: https-gradle
```

**Review by thepetk — CHANGES_REQUESTED** [review]

**Comment by dkwon17** [comment]
Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit

**Comment by thepetk** [comment]
> Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit

Thanks! @ajm01 as you are the owner of the stack could you please review the PR?

/hold

**Comment by thepetk** [comment]
Note there's a known issue with a failing check and we're working to address it: https://github.com/devfile/api/issues/1494.

**Comment by thepetk** [comment]
> > Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit
> 
> Thanks! @ajm01 as you are the owner of the stack could you please review the PR?
> 
> /hold

@ajm01 kind reminder to review the PR as you are listed as an owner. As discussed in the devfiles community call, we will wait for a week, until 23 Apr 2024, so the owner has time to review. In case we don't have a review from the owner, the devfile services team will review & merge the updates to unblock the `eclipse che` team.

**Review by thepetk — APPROVED** [review]
/lgtm

**Comment by thepetk** [comment]
> > > Thank you, I fixed the changes, but I had to force push to my branch to remove the unsigned commit
> > 
> > 
> > Thanks! @ajm01 as you are the owner of the stack could you please review the PR?
> > /hold
> 
> @ajm01 kind reminder to review the PR as you are listed as an owner. As discussed in the devfiles community call, we will wait for a week, until 23 Apr 2024, so the owner has time to review. In case we don't have a review from the owner, the devfile services team will review & merge the updates to unblock the `eclipse che` team.

Merged

**Pull request merged**
